cloudsim-plus-examples/src/main/java/org/cloudsimplus/examples/brokers/CloudletToVmMappingBestFit.java

```diff
@@ -128,13 +128,21 @@ public CloudletToVmMappingBestFit() {
      * @return the VM selected for the Cloudlet or {@link Vm#NULL} if no suitable VM was found
      */
     private Vm bestFitCloudletToVmMapper(final Cloudlet cloudlet) {
+        if (cloudlet.isBindToVm()) {
+            if(broker0.equals(cloudlet.getVm().getBroker()) && cloudlet.getVm().isCreated()) {
+                return cloudlet.getVm();
+            }
+
+            return Vm.NULL;
```

Comment:
Maybe if the Cloudlet is bound to a VM but that VM is not created yet, we should just let the best-fit policy try to find a VM, by removing this return Vm.NULL. What do you think?

Reply:
Sounds reasonable. I made the change.